src_new/adapters/prebid.js

```diff
@@ -389,10 +389,11 @@ function generatedKeyCallback(adapterID, adUnits, adapterConfig, impressionID, g
 			break;
 
 		case "pubmatic":
+		case "pubmatic2":
 			slotParams["publisherId"] = adapterConfig["publisherId"];
 			slotParams["adSlot"] = generatedKey;
```

Comment:
Should we send slotname instead of generatedKey for pubmatic2 ? 
Secondary publisher should map that slotname with pubmatic siteid and tag id in his profile